app/controllers/triannon/auth_controller.rb

```diff
@@ -5,64 +5,56 @@ module Triannon
   class AuthController < ApplicationController
     include RdfResponseFormats
 
-    AUTH_EXPIRY  =   60 # seconds
-    TOKEN_EXPIRY = 3600 # seconds
+    # HTTP request methods accepted by /auth/login
+    LOGIN_ACCEPT = 'OPTIONS, GET, POST'
 
     # OPTIONS /auth/login
     def options
```

Comment:
OPTIONS is required to make CORS work.  it's called a "pre-flight request" and a stack overflow question on it that's pretty good: http://stackoverflow.com/questions/8685678/cors-how-do-preflight-an-httprequest 
The only requirement for it is that it returns the CORS headers. No need for there to be a body at all.